src/jit/emitxarch.cpp

```diff
@@ -5360,12 +5455,85 @@ void emitter::emitIns_SIMD_R_R_I(instruction ins, emitAttr attr, regNumber reg,
     }
 }
 
+void emitter::emitIns_SIMD_R_R_R_A(
+    instruction ins, emitAttr attr, regNumber reg, regNumber reg1, regNumber reg2, GenTreeIndir* indir)
```

Comment:
In this (and subsequent), I would name the first reg argument `targetReg` or `dstReg` or something. Unlike some of the other methods, this one is designed only to support the case where the first argument is the dest, so it would be good to be descrptive.

Reply:
Can I submit a separate PR fixing up all the register names here? Currently the majority are `regNumber reg` where it should be `regNumber targetReg` and it would be nice to fix them all up (I could also do it in this PR, if you think that is fine).